machine_learning_hep/computetrigger.py

```diff
@@ -12,110 +12,225 @@
+    hratiod.Fit(funcd, "L", "", *fit_range)
+    func.SetLineColor(1)
+    func.Draw("same")
+    funcd.SetLineColor(4)
+    funcd.Draw("same")
+    # Draw both fitting functions
+    ctrigger.cd(6)
+    # pylint: disable=unused-variable
+    hframe = gPad.DrawFrame(min_draw_range, 0,
+                            max_draw_range, 1,
+                            ";n_tracklets_corr;Efficiency")
+    funcnorm = func.Clone("funcSPDvspt_ntrkl_norm")
+    funcnorm.FixParameter(0, funcnorm.GetParameter(0)/funcnorm.GetMaximum())
+    funcnormd = funcd.Clone("funcdSPDvspt_ntrkl_norm")
+    funcnormd.FixParameter(0, funcnormd.GetParameter(0)/funcnormd.GetMaximum())
+    funcnorm.Draw("same")
+    funcnormd.Draw("same")
+    line = TLine(60, 0, 60, 1)
+    line.SetLineStyle(2)
+    line.Draw("same")
+    ctrigger.SaveAs(output_path + "/SPDtrigger.pdf")
+    foutput = TFile.Open(output_path + "/triggerSPDvspt_ntrkl.root",
+                         "recreate")
+    foutput.cd()
+    hratio.SetName("hratioSPDvspt_ntrkl")
+    hratio.Write()
+    func.SetName("funcSPDvspt_ntrkl")
+    func.Write()
+    funcnorm.Write()
+    hratiod.SetName("hratiodSPDvspt_ntrkl")
+    hratiod.Write()
+    funcd.SetName("funcdSPDvspt_ntrkl")
+    funcd.Write()
+    funcnormd.Write()
+    if "-b" not in argv:
+        print("Press enter to continue")
+        input()
+    foutput.Close()
+
+
+if __name__ == "__main__":
+    # Configuration variables
+    PARSER = argparse.ArgumentParser(description="Compute the trigger")
+    PARSER.add_argument("--input_trg",
+                        help="input file for triggered data")
+    PARSER.add_argument("--input_mb",
+                        help="input file for MB data")
+    PARSER.add_argument("--output_path",
+                        help="output path for pdf and root files",
+                        default="/tmp/")
+    PARSER.add_argument("--min_draw_range",
+                        help="Minimum histogram plotting range",
+                        default=0.,
+                        type=float)
+    PARSER.add_argument("--max_draw_range",
+                        help="Maximum histogram plotting range",
+                        default=150.,
+                        type=float)
+    PARSER.add_argument("--min_fit_range",
+                        help="Minimum fit range",
+                        default=40.,
+                        type=float)
+    PARSER.add_argument("--max_fit_range",
+                        help="Maximum fit range",
+                        default=100.,
+                        type=float)
+    PARSER.add_argument("--rebin_histo",
+                        help="Rebin the histogram",
+                        default=True,
+                        type=bool)
+    PARSER.add_argument("--show_func_ratio",
+                        help="Shows the ratio between the function and the fitted histogram",
+                        default=True,
+                        type=bool)
+    PARSER.add_argument("-b",
```

Comment:
Where do you use this?

Reply:
Line 179

Reply:
I reckon it would be better to avoid unrecognised arguments, but this way it is too broad and you could include also unwanted/unsupported options